External/Plugins/ProjectManager/Controls/OpenResourceForm.cs

```diff
@@ -492,45 +497,72 @@ public static List<string> getMatchedItems(List<string> source, string searchTex
             try
             {
                 searchFile = Path.GetFileName(searchText);
-                searchDir = Path.GetDirectoryName(searchText);
+                searchDir = Path.GetDirectoryName(searchText);

```

Comment:
`Path.GetDirectoryName(searchText)` automatically converts `/` to `\\` (on Windows), so searchDir has `\\`, meanwhile `searchText` had `/`